website/style.css

```diff
@@ -15,16 +15,31 @@ body {
 }
 
 header {
-  padding: 24px 0px 24px 0px;
+  padding: 48px 0px 32px 0px;
   background-color: white;
 }
 
 .inside-header {
   padding: 0px 32px 0px 32px;
+  display: flex;
+}
+
+.header-content{
+  width: fit-content;
+}
+
+.header-logo {
+  margin-right: 48px;
+}
+
+.ecosystem-title {
+  font-size: 60px;
```

Comment:
I think this should probably use `rem`: https://carbondesignsystem.com/guidelines/typography/overview/#scale